filebeat/tests/integration/testdata/filebeat_system_module.yml

```diff
@@ -14,3 +18,4 @@ output:
   file:
     path: ${path.home}
     filename: "output"
+    rotate_every_kb: 62500 # 500Mb
```

Comment:
Is this every 62500kb? If so that is 62,5MB and not 500. Did I miss something?

Reply:
I didn't pay attention to the bit/byte difference there :man_facepalming: 
![2024-10-23_08-49](https://github.com/user-attachments/assets/93434932-d8d8-4afc-bda4-5acbb4d649c5)

Reply:
It happens to the best of us! Thanks for addressing.